rows.go

```diff
@@ -57,7 +58,9 @@ type rows struct {
 	tzOffset      string
 	inMemRowLimit int
 	resultCache   *os.File
+	ReadWriteBuf  *bufio.ReadWriter
 	cachingFailed bool
+	scratch       [512]byte
```

Comment:
Any reason it's 512?

Reply:
No. I don't really have an idea what a good size could be.